project/UI/Markdown/main.swift

```diff
@@ -0,0 +1,46 @@
+//  Copyright © 2019 The nef Authors.
+
+import Foundation
+import CLIKit
+import nef
+import Bow
+import BowEffects
+
+private let console = Console(script: "nef-markdown",
```

Comment:
`Console` here should be renamed as `Command` or something like that. We are using the term `Console` in other context with a different meaning (the place where we are printing stuff) and it causes confusion.